.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Version and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      newversion:
+        description: 'Version'
+        required: true
+
+concurrency:
+  group: npm-bump
+  cancel-in-progress: true
+
+jobs:
+  version_and_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        registry-url: 'https://registry.npmjs.org'
```

Comment:
I think this is optional and the default value already.

Reply:
The docs state that it is required for publishing: https://github.com/bcomnes/npm-bump